Automattic/woocommerce-payments
# Remove user facing mentions to Prototype from the new Onboarding flow

Fixes #6298 

#### Changes proposed in this Pull Request

This PR removes the "Prototype" mention in the new flow title, and also modifies the path in that new flow from `onboarding-prototype` to `onboarding-flow`.

#### Testing instructions

* Enable Progressive Onboarding to see the new flow
* Onboard a new account
* Make sure the title of the new page is "Onboarding" and not "Onboarding Prototype"
* Check the path of the page and make sure it's `onboarding-flow` and not `onboarding-prototype`.
* Make sure the whole onboarding works as expected.

-------------------


**Post merge**
